sound/ambience/antag/attribution.txt

```diff
@@ -0,0 +1,5 @@
+{
+abductee.ogg
+hypnotized.ogg
+brainwashed.ogg
+} - from https://github.com/BeeStation/BeeStation-Hornet/pull/9759
```

Comment:
```suggestion
sound/ambience/antag/abductee.ogg is from "Warp SFX" https://freesound.org/people/Breviceps/sounds/453391 (CC0)
sound/ambience/antag/brainwash.ogg is from "nog.wav" https://freesound.org/people/_NOMINAL_/sounds/124602 (CC-BY 3.0)
sound/ambience/antag/hypnosis.ogg is from "Flashback.wav" https://freesound.org/people/Sclolex/sounds/342103 (CC0)
```
These are the actual attributions, taken from the Beestation PR.

Reply:
oh yes, thanks!